kolibri/core/assets/src/views/content-renderer/index.vue

```diff
@@ -86,6 +89,17 @@
         type: Function,
         default: () => Promise.resolve(),
       },
+      // Allow content renderers to display in a static mode
+      // where user interaction is not allowed
+      interactive: {
+        type: Boolean,
+        default: true,
+      },
```

Comment:
cool. How was this handled before?

Reply:
It wasn't - on the coach reports, coaches could seemingly change students answers (but they couldn't really, so it was just misleading).